docs/maintenance-tasks.md

```diff
@@ -2,48 +2,63 @@
 * Command: `sudo zpool scrub nest`
 
-To execute manually, you must first ssh into hen from rooster with the command `ssh hen`. Scrub checks the file system's integrity, and repairs any issues that it finds. After the scrub is finished, it is good to also run `zpool status` to check if there is anything wrong.
-
-This command is run on a cronjob, so there should be no need for manual intervention. The cronjob runs at 8:00AM the first day of every month. Rooster will also send out an email at 8:10AM on the same day with the results. If the scrub is fine, the email should be titled `[All clear] Hen monthly ZFS report`. If the title says `[POTENTIAL ZFS ISSUE]` instead, there may be something wrong with the disk, and the email contains the `zpool status` output which you can use to debug disk issues. Details on how the cronjob is setup are in the private configuration repo, under `cronjob/monthly-zfs-report.py`.
+To execute manually, you must first ssh into blackhole 
+Scrub checks the file system's integrity, and repairs 
+any issues that it finds. After the scrub is finished, 
+it is good to also run `zpool status` to check if there 
+is anything wrong.
+
+This command was previously run on a cronjob. The cronjob ran at 8:00AM 
+the first day of every month. Gravity would also send out an 
+email at 8:10AM on the same day with the results. If the 
+scrub is fine, the email would be titled 
+`[All clear] Hen monthly ZFS report`. If the title 
+says `[POTENTIAL ZFS ISSUE]` instead, there may be 
+something wrong with the disk, and the email contains 
+the `zpool status` output which can be used to debug disk 
+issues. Details on how the cronjob was setup are in the 
+private configuration repo, under `cronjob/monthly-zfs-report.py`.
 
 ## Cluster control plane upgrade
 
-The Kubernetes control plane should be upgraded regularly. There is a cronjob sending out a triyearly reminder (Jan, May, Sept 1st of every year) reminding you to do the upgrade. (The cronjob can be found in the private configuration repo, under `cronjob/cluster-upgrade-reminder.sh).
-
-This must be done at least once a year, otherwise the Kubernetes certificates may expire, which will [break the entire cluster](https://github.com/LibreTexts/metalc/blob/master/docs/Bare-Metal/troubleshooting/KubeadmCert.md#more-complex-solution-renewing-kubeadm-certificates). The email sent out should contain the date on which the certificates expire. If you apply routine upgrades, certificates should be renewed automatically and this should not be an issue. If the certificates do expire, follow that guide instead of this one. If you need to renew the certificates without upgrading the cluster, follow [the guide to renew certificates without upgrading](#renew-certificates-without-upgrade) instead, but this should not be an excuse to put off cluster upgrades indefinitely.
+The Kubernetes control plane should be upgraded regularly. 
+There used to be a cronjob sending out a triyearly reminder 
```

Comment:
Cronjob also still in place so far as i know. If you are trying to explicitly talk about flock, maybe just "flock had" rather than "there used to be", which implies that there's no similar system in place

Reply:
Fixed